builds/misc/images.yaml

```diff
@@ -35,91 +35,111 @@ jobs:
         name: build
         displayName: Build ($(Build.Configuration))
 
-      - script: scripts/linux/buildDiagnostics.sh $(Build.Configuration)
-        displayName: Build iotedge-diagnostics
+      - script: scripts/linux/buildEdgelet.sh -i azureiotedge-iotedged -n microsoft -P iotedged -c $(Build.Configuration) -t aarch64
+        displayName: Build - Edge Security Daemon - aarch64
+
+      - script: scripts/linux/buildEdgelet.sh -i azureiotedge-diagnostics -n microsoft -P iotedge-diagnostics -c $(Build.Configuration) -t aarch64
+        displayName: Build - Edge Diagnostics - aarch64
+
+      - script: scripts/linux/buildEdgeProxy.sh -i azureiotedge-proxy -n microsoft -P edge-proxy -t aarch64
+        displayName: Build - Edge Proxy - aarch64
 
       - task: PublishBuildArtifacts@1
         displayName: 'Publish Artifacts'
         inputs:
           PathtoPublish: '$(Build.BinariesDirectory)/publish'
           ArtifactName: 'core-linux'
 
-     # azureiotedge-diagnostics - Not Using Template for ARM64 because we have 2 different .NET Core.
+      # Edge Diagnostics - Not Using Template for ARM64 because we have 2 different .NET Core.
       - task: Bash@3
-        displayName: Build Image azureiotedge-diagnostics - aarch64
-        inputs: 
-           filePath: scripts/linux/buildImage.sh
-           arguments: -r $(registry.address) -u $(registry.user) -p $(registry.password) -i azureiotedge-diagnostics -n microsoft -P azureiotedge-diagnostics --target-arch aarch64
+        displayName: Build Image - Edge Security Daemon - aarch64
```

Comment:
why are we building this new docker image for iotedge security daemon?  it is built in edgelet package build and should be run on the host, not docker container.

Reply:
Yes, you are right. In "normal mode" it runs as a native service on the host. But we are working on adding a new mode that allows edge run on Kubernetes. To support it, we need to package security daemon as a docker image.